### PR TITLE
Fix: Pin Rubocop version to 2.0.0+

### PR DESCRIPTION
**Pull request by jk2l**
Pin Rubocop version to 2.0.0+ for dependency fix

### CODE REVIEW

**Comment by bastelfreak** [comment]
LGTM. I would like to merge it here + open up a PR in modulesync_config so we can put the fix on all modules.

File: Rakefile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
-RuboCop::RakeTask.new(:rubocop) do |task|
-  # These make the rubocop experience maybe slightly less terrible
-  task.options = ['-D', '-S', '-E']
+if RUBY_VERSION >= '2.0.0'
+  require 'rubocop/rake_task'
+
+  RuboCop::RakeTask.new(:rubocop) do |task|
+    # These make the rubocop experience maybe slightly less terrible
+    task.options = ['-D', '-S', '-E']
```
Comment (igalic) [inline Rakefile]: :+1:

File: Gemfile [hunk 2]
```diff
@@ -26,7 +26,8 @@ group :test do
   gem 'puppet-blacksmith',                                          :require => false, :git => 'https://github.com/voxpupuli/puppet-blacksmith.git'
   gem 'voxpupuli-release',                                          :require => false, :git => 'https://github.com/voxpupuli/voxpupuli-release-gem.git'
   gem 'puppet-strings',                                             :require => false, :git => 'https://github.com/puppetlabs/puppetlabs-strings.git'
-  gem 'rubocop-rspec', '~> 1.5',                                    :require => false
+  gem 'json_pure', '<= 2.0.1',                                      :require => false if RUBY_VERSION < '2.0.0'
+  gem 'rubocop-rspec', '~> 1.5',                                    :require => false if RUBY_VERSION >= '2.0.0'
```
Comment (igalic) [inline Gemfile]: this seems sensible.

File: Gemfile [hunk 3]
```diff
@@ -26,7 +26,8 @@ group :test do
   gem 'puppet-blacksmith',                                          :require => false, :git => 'https://github.com/voxpupuli/puppet-blacksmith.git'
```
Comment (igalic) [inline Gemfile]: didn't we release a new version here?

Comment (bastelfreak) [inline Gemfile]: we did. I will fix that in modulesync.

**Pull request merged**